testsuite/run_sets/testsuite.yml

```diff
@@ -22,18 +22,18 @@
   # one of: core_proxy_register_as_trad_with_script.feature
   #         core_proxy_register_as_minion_with_script.feature
   #         core_proxy_register_as_minion_with_gui.feature
-- features/core_proxy_register_as_minion_with_gui.feature
-- features/core_proxy_branch_network.feature
+#- features/core_proxy_register_as_minion_with_gui.feature
+#- features/core_proxy_branch_network.feature
 # initialize clients
-- features/core_trad_register_client.feature
-- features/core_min_bootstrap.feature
-- features/core_min_salt_ssh.feature
-- features/core_centos_tradclient.feature
-- features/core_centos_salt.feature
-- features/min_centos_salt_install_package_and_patch.feature
-- features/core_centos_salt_ssh.feature
-#- features/core_ubuntu_salt.feature
-#- features/core_ubuntu_salt_ssh.feature
```

Comment:
only one ubuntu feature in "core" - see with @lkotek

Reply:
Let's address it another PR, so it's easier to merge that one in ubuntu-support branch.
Remember that PR is not going to Manager-3.2, but to ubuntu-support branch (dev branch)